bundles/org.openhab.ui/web/src/assets/automation-languages.js

```diff
@@ -1,12 +1,28 @@
-export const AddonNames = {
-  'application/javascript': 'JavaScript Scripting',
-  'application/javascript;version=ECMAScript-5.1': 'JavaScript Scripting (Nashorn)',
-  'application/x-groovy': 'Groovy Scripting',
-  'application/x-python': 'Jython Scripting',
-  'application/x-ruby': 'JRuby Scripting'
-}
-
-export const DocumentationLinks = {
-  'application/javascript': '/link/jsscripting-script',
-  'application/x-ruby': '/link/jrubyscripting-script'
+export const AUTOMATION_LANGUAGES = {
+  'application/javascript': {
+    'name': 'JavaScript Scripting',
```

Comment:
@florian-h05 what's the generally preferred syntax for defining objects? i.e. I would prefer

```javascript
export const AUTOMATION_LANGUAGES = {
  'application/javascript': {
  name: '......',
  commentChar: '....'
}}
```

instead of `'name': '....'` etc.

Also `ObjectName.key1.subkey` as opposed to `ObjectName['key1']['subkey']`

Reply:
I prefer and I think its common to not use quotes as long as they aren‘t required, like in your example.
When accessing object properties, you usually use the dot notation, the `[string]` notation is only used if dot notation won‘t work due to special chars or you need to use vars.

I would have requested to change that when doing a full review, just had a swift look at the code.